src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Msmq/MessageQueue_Purge_Integration.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Msmq
+{
+    /// <summary>
+    /// Msmq calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+    AssemblyName = "System.Messaging",
+    TypeName = "System.Messaging.MessageQueue",
+    MethodName = "Purge",
+    ReturnTypeName = ClrNames.Void,
+    MinimumVersion = "4.0.0",
+    MaximumVersion = "4.*.*",
+    IntegrationName = IntegrationName)]
+    public class MessageQueue_Purge_Integration
+    {
+        private const string Command = "msmq.purge";
+        private const string IntegrationName = nameof(IntegrationIds.Msmq);
```

Comment:
Can you reuse the value in `MsmqConstants.IntegrationName`? Consider also moving `"msmq.purge"` there.